test/jest/roktManager.spec.ts

```diff
@@ -254,7 +277,12 @@ describe('RoktManager', () => {
                     ])
                 },
             };
-            roktManager.init(kitConfig as IKitConfigs, {} as IMParticleUser, mockMPInstance.Identity);
+            roktManager.init(
+                kitConfig as IKitConfigs,
+                {} as IMParticleUser,
+                mockMPInstance.Identity,
+                mockMPInstance._Store
+            );
```

Comment:
Assume this was just missed here

```suggestion
                mockMPInstance._Store,
                mockMPInstance.Logger,
            );
```